neo/Plugins/Plugin.cs

```diff
@@ -39,23 +44,60 @@ public static bool CheckPolicy(Transaction tx)
 
         internal static void LoadPlugins(NeoSystem system)
         {
+            if (_configWatcher != null)
```

Comment:
Why is `_configWatcher != null` before `LoadPlugins()`?

Reply:
This line is to prevent multiple calls to `LoadPlugins`